pkg/config/config.go

```diff
@@ -635,7 +635,7 @@ func (lci *LocalConfigInfo) GetOSSourcePath() (path string, err error) {
 	componentContext := strings.Trim(filepath.Dir(lci.Filename), ".odo")
 
 	if sourceLocation == "" {
-		return "", fmt.Errorf("Blank source location provided")
+		return "", fmt.Errorf("Blank source location, does the .odo directory exist?")
```

Comment:
`check if .odo directory exists?` <- sounds better?

Reply:
IMO no, but it conveys the same. I don't mind either way. `check if the .odo directory exists` sounds like too many words :(

Reply:
I liked this one `does the .odo directory exist`